src-electron/db/query-command.js

```diff
@@ -1504,13 +1504,15 @@ async function selectCommandDetailsFromAllEndpointTypesAndClusters(
     C.CLUSTER_REF = CLUSTER.CLUSTER_ID
   WHERE
     ETC.ENDPOINT_TYPE_REF IN (${endpointTypeIds})
-    AND ETC.ENDPOINT_TYPE_CLUSTER_REF in (${endpointClusterIds}) `
+    AND ETC.ENDPOINT_TYPE_CLUSTER_REF in (${endpointClusterIds})`
 
   if (doGroupBy) {
     // See: https://github.com/project-chip/zap/issues/192
-    query = query + ` GROUP BY C.NAME, C.COMMAND_ID`
+    query = query + ' GROUP BY C.NAME, C.COMMAND_ID'
   }
 
+  query = query + ' ORDER BY C.MANUFACTURER_CODE, C.CODE, C.NAME'
```

Comment:
To keep the sort closer to what we were accidentally/usually getting out of the group by, would it make sense to sort on NAME first?

I worry about the major changes to codegen that will likely happen otherwise when this is imported into Matter, which will make it very hard to review whether the changes are intended...

If we _do_ want to change the sort order significantly, I'd like it if it happened in a separate PR so there is a separate changeset we do the Matter update to so we can easily check that all the codegen changes are just from the ordering change.